test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/RestoreCommandTests.cs

```diff
@@ -500,6 +505,68 @@ public async Task Restore_LegacyPackagesConfig_WithNuGetLockFileLockedMode()
             }
         }
 
+        [Fact]
+        public async Task RestorePackagesConfig_WithExistingLockFile_LockedMode_Succeeds()
```

Comment:
```suggestion
        public async Task RestorePackagesConfig_WithExistingLockFile_LockedMode_Succeeds_Async()
```

Reply:
Async is usually only used when an API is public, so tests wouldn't fall under it. 

If you look at the test case names, it's very much split. 
We can try to define a rule for it(feel free to champion it) but for now, I think it's ok to just keep it as is. In fact I actually prefer it this way :)